subprojects/plugins/src/main/java/org/gradle/api/plugins/TestReportAggregationPlugin.java

```diff
@@ -72,26 +77,28 @@ public void apply(Project project) {
             testReportDirectory.convention(javaPluginExtension.getTestReportDir());
         });
 
-        // iterate and configure each user-specified report, creating a <reportName>Results configuration for each
-        reporting.getReports().withType(AggregateTestReport.class).all(report -> {
-            // A resolvable configuration to collect test results; typically named "testResults"
-            Configuration testResultsConf = project.getConfigurations().create(report.getName() + "Results");
-            testResultsConf.extendsFrom(testAggregation);
-            testResultsConf.setDescription(String.format("Supplies test result data to the %s.  External library dependencies may appear as resolution failures, but this is expected behavior.", report.getName()));
-            testResultsConf.setVisible(false);
-            testResultsConf.setCanBeConsumed(false);
-            testResultsConf.setCanBeResolved(true);
-            testResultsConf.attributes(attributes -> {
-                attributes.attribute(Category.CATEGORY_ATTRIBUTE, objects.named(Category.class, Category.VERIFICATION));
-                attributes.attributeProvider(TestSuiteType.TEST_SUITE_TYPE_ATTRIBUTE, report.getTestType().map(tt -> objects.named(TestSuiteType.class, tt)));
-                attributes.attribute(VerificationType.VERIFICATION_TYPE_ATTRIBUTE, objects.named(VerificationType.class, VerificationType.TEST_RESULTS));
-            });
+        // A resolvable configuration to collect test results
+        Configuration testResultsConf = project.getConfigurations().create("aggregateTestReportResults");
+        testResultsConf.extendsFrom(testAggregation);
+        testResultsConf.setDescription("Supplies test result data. External library dependencies may appear as resolution failures, but this is expected behavior.");
+        testResultsConf.setVisible(false);
+        testResultsConf.setCanBeConsumed(false);
+        testResultsConf.setCanBeResolved(true);
+        getEcosystemUtilities().configureAsRuntimeClasspath(testAggregation);
```

Comment:
Technically, we may only want to do this when the java-base plugin is applied, but I'm not sure.

Reply:
Updated to only add the runtimeClasspath attributes if the current project applies the java base pluign. This restricts sorta to only aggregating a single ecosystem at once. 

If the aggregating project does not declare the java-base plugin, this resolvable will have no attributes and we'll rely on fallback behavior (which may actually allow us to do multi-ecosystem aggragation).